lib/software_challenge_client/coordinates.rb

```diff
@@ -10,6 +10,10 @@ def initialize(x, y)
   end
 
   def ==(other)
+    eql?(other)
+  end
+
+  def eql?(other)
```

Comment:
Wozu genau ist die eql methode?
Laut VScode wird sie nur ein mal in der == def benutzt 
![image](https://user-images.githubusercontent.com/19777592/98740024-aa175380-23aa-11eb-80b6-c46933381ace.png)

Reply:
Ich hatte versucht, Sets zum laufen zu bekommen.
die doku sagte, Sets nutzen eql? zum Vergleich.
Aber es hat nicht funktioniert, also... es ist redundant

Reply:
Okay das ändere ich sonst gleich mit dem Hinzufügen der Doku